### PR TITLE
added messages endpoint

### PR DESCRIPTION
**Pull request by vshmyhlo**

### CODE REVIEW

File: app/serializers/vertigo/rtm/message_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  module Rtm
+    class MessageSerializer < ApplicationSerializer
+      attributes :text,
+                 :creator_id,
+                 :conversation_id,
+                 :created_at,
+                 :updated_at
+
+      attribute :created_at do
+        object.created_at.to_s(:db)
```
Comment (RomanMukhin) [inline app/serializers/vertigo/rtm/message_serializer.rb]: It's recommended to use ISO8601 by JSON API recommendations http://jsonapi.org/recommendations/#date-and-time-fields . I suggest to use it in future.

time_variable.to_formatted_s(:iso8601)

File: app/serializers/vertigo/rtm/message_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+module Vertigo
+  module Rtm
+    class MessageSerializer < ApplicationSerializer
+      attributes :text,
+                 :creator_id,
+                 :conversation_id,
+                 :created_at,
```
Comment (RomanMukhin) [inline app/serializers/vertigo/rtm/message_serializer.rb]: We don't need to specify 'created_at' and 'updated_at' in 'attributes', if they are specified in 'attribute'.

File: spec/controllers/vertigo/rtm/messages_controller_spec.rb [hunk 3]
```diff
@@ -3,16 +3,235 @@
...
+            it 'responds with resource json' do
+              perform_request
+              expect(json_response[:data][:attributes][:text]).to eq(message_params[:text])
+            end
+          end
+
+          context 'with invalid params' do
+            let(:message_params) { invalid_message_params }
+
+            it 'responds with errors json' do
```
Comment (RomanMukhin) [inline spec/controllers/vertigo/rtm/messages_controller_spec.rb]: This test looks the same as https://github.com/rubygarage/vertigo-rtm/pull/11/files#diff-2bcfc7d38c4d8a4e11403dd52acc51feR126

**Comment by GeorgeZhukov** [comment]
ok

**Comment by RomanMukhin** [comment]
+

**Comment by kirillshevch** [comment]
👍

**Pull request merged**